django_recaptcha/apps.py

```diff
@@ -1,11 +1,11 @@
 from django.apps import AppConfig
 from django.core.checks import Tags, register
 
-from captcha.checks import recaptcha_key_check
+from django_recaptcha.checks import recaptcha_key_check
 
 
-class CaptchaConfig(AppConfig):
-    name = "captcha"
+class DjangoRecaptchaConfig(AppConfig):
+    name = "django_recaptcha"
```

Comment:
This should stay the same to avoid package conflict

Reply:
Hey @Andrew-Chen-Wang, shouldn't this be the same as the namespace? [The Django docs suggest this should be the full Python path to the package](https://docs.djangoproject.com/en/4.2/ref/applications/#django.apps.AppConfig.name)

If not for technical reason, I still think it should change so the naming is consistent. Won't you agree? 

Also, I'm not sure what package conflict you are referring to. Could you elaborate, please?

Reply:
Sorry, I meant django_recaptcha is correct.